### PR TITLE
Initial work on kubernetes-workshop basics

### PR DESCRIPTION
**Pull request by tongpu**
This is the base for a workshop on Kubernetes basics.

### CODE REVIEW

File: kubernetes-workshop/6001_basics/modul_basics_6001_01_slides.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+    servicePort: 80
+  tls:
+    - hosts:
+      - "echo.example.com"
+      secretName: "echo-tls"
+```
+
+## Replication Controller
+
+Manages a set of pods
```
Comment (karras) [inline kubernetes-workshop/6001_basics/modul_basics_6001_01_slides.md]: s/pods/Pods/

File: kubernetes-workshop/6001_basics/modul_basics_6001_01_slides.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+* configurable number of Replicas
```
Comment (karras) [inline kubernetes-workshop/6001_basics/modul_basics_6001_01_slides.md]: s/configurable/Configurable/

File: kubernetes-workshop/6001_basics/modul_basics_6001_01_slides.md [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+* even useful when `replicas=1`
```
Comment (karras) [inline kubernetes-workshop/6001_basics/modul_basics_6001_01_slides.md]: s/even/Even/

File: kubernetes-workshop/6001_basics/modul_basics_6001_01_slides.md [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+Will be replaced by Deployments/ReplicaSets
+
+## Deploymentᵝ
+
+De facto standard for deploying applications
+
+* manages multiple iterations of an application
+* based on ReplicaSets
+* allows rollback of deployments
+
```
Comment (karras) [inline kubernetes-workshop/6001_basics/modul_basics_6001_01_slides.md]: To be consistent, the first words should be uppercase again.

File: kubernetes-workshop/6001_basics/modul_basics_6001_01_slides.md [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+- Automatic provisioning in Azure Cloud
+- Different requirements via StorageClass
+
+## Namespace
+
+Separation of costumers/environments
+
+* Ressource Quotas
+* all resources are attached to a namespace
```
Comment (karras) [inline kubernetes-workshop/6001_basics/modul_basics_6001_01_slides.md]: s/all/All/

File: kubernetes-workshop/6001_basics/modul_basics_6001_01_slides.md [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+kubectl create -f http://example.com/pod.yaml
+cat pod.yaml | kubectl create -f -
+kubectl create -R -f dir/
+```
+
+## kubectl get
+
+List Kubernetes resources
+
+* differnt output formats
```
Comment (karras) [inline kubernetes-workshop/6001_basics/modul_basics_6001_01_slides.md]: s/differnt/Different/

File: kubernetes-workshop/6001_basics/modul_basics_6001_01_slides.md [hunk 7]
```diff
@@ -0,0 +1,198 @@
...
+- CLI for **everything**
+- short for kube control
+- [some](https://twitter.com/search?q=kube-cuddle) believe it is pronounced kube-cuddle
+
+## kubectl create
+
+Create resources in Kubernetes
+
+* Definition in YAML or JSON
+* Validation in `kubectl` Client
```
Comment (karras) [inline kubernetes-workshop/6001_basics/modul_basics_6001_01_slides.md]: s/Client/client/

File: kubernetes-workshop/6001_basics/modul_basics_6001.yml [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+
+Description: >
+    This workshop will provide you with a basic introduction to Kubernetes and
+    allows you to deepen you knowledge with future workshops.
+
+Requirements:
+    - Linux Distribution
+    - Bash shell
+    - minikube
+    - kubectl
```
Comment (tongpu) [inline kubernetes-workshop/6001_basics/modul_basics_6001.yml]: We should think about that. But I would recommend to specify a standard and update all YAML files accordingly.

**Pull request merged**